src/v2/Components/Artwork/Details.tsx

```diff
@@ -1,10 +1,15 @@
-import { Flex, Link, Text } from "@artsy/palette"
```

Comment:
Realize this is gonna be hard to review. Basically just converted it to functional without any major changes to logic or structure.